release.py

```diff
@@ -62,8 +62,10 @@ def create_data_schema_documentation():
 
 def make_release_specific_changes(release_version):
     execute_steps([
-        "python ./web_frontend/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
-        "python ./osmaxx_conversion_service/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
+        "python ./web_frontend/manage.py makemessages -l en",
+        "python ./osmaxx_conversion_service/manage.py makemessages -l en",
```

Comment:
Shouldn't these two commands produce one `*.po` file each, thus two in total? [Only one](https://github.com/geometalab/osmaxx/pull/798/files#diff-36e525b0fdeb572e17dd3158ee2245e8) seems to remain. Are they overwriting each other's output?